src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -428,6 +466,10 @@ PerfAnalyzer::WriteReport()
   bool should_output_metrics{
       params_->should_collect_metrics && params_->verbose_csv};
 
+  // TODO (TMA-1526): Detect if the model is LLM and report LLM metrics based
```

Comment:
Do we need to know if it is llm? Can't we report metrics based on if we get multiple responses for a request ? (token to token, first response latency, etc.) - seem generic?

Reply:
Tokens are a concept specific to LLMs no?
Users benchmarking decoupled models that do not use tokenizers would likely be confused if those metrics were reported.

Reply:
I thought there were at least 3 cases where we would want this information:
- What metrics to output
- What stimulus to send in by default (better LLM default stimulus)
- What metrics to stabilize on

I wouldn't say it is required, but my hope was to have it implemented and abstracted away so we didn't have multiple places of code trying to figure out their own way to make this distinction.